src/_internals/itineraryLocation/ItineraryLocation.tsx

```diff
@@ -134,6 +138,12 @@ export const ItineraryLocation = ({
     return renderSubLabel(subLabel)
   }
 
+  const refConnection = useRef<HTMLDivElement>(null)
+  const [height, setHeight] = useState(0)
+  useEffect(() => {
```

Comment:
As `useEffect` is called _some times after_ the update is committed to the DOM, there might be some visual glitch. You can use [`useLayoutEffect`](https://reactjs.org/docs/hooks-reference.html#uselayouteffect) which is called right after the DOM update.

> Use this to read layout from the DOM and synchronously re-render. Updates scheduled inside `useLayoutEffect` will be flushed synchronously, before the browser has a chance to paint.
> https://reactjs.org/docs/hooks-reference.html#uselayouteffect

Reply:
Nice!